src/util/value.ts

```diff
@@ -24,6 +24,8 @@ export type Camera = {
   fov: number;
 };
 
+export type Clock = CesiumClock | undefined;
```

Comment:
This file is typically for value types and easy conversion between typescript types and GQL value types (please see below code, especially the comment stating any new field should be added to valueTypeMapper). Unless I am mistaken, perhaps you should move this type somewhere else! If it is needed for converting between GQL types as well, please follow the comment's instructions

Reply:
Also, I think it is clearer to have this typed as `export type Clock = CesiumClock` and then where you use it state it can be `undefined`.